sprintf dependency not detected/installed correctly

```
 ./bin/hubot 
Loading deploy-local scripts at /home/leto/git/hubot/scripts
Loading hubot core scripts for relative scripts at /home/leto/git/hubot/src/hubot/scripts
Hubot: the Shell.
/^Hubot:?\s*show (me )?builds/i
/^Hubot:?\s*what's playing (.*)/i
/^Hubot:?\s*Y U NO (.+)/i
/^Hubot:?\s*(I DON'?T ALWAYS .*) (BUT WHEN I DO .*)/i
/^Hubot:?\s*(.*)(O\s?RLY\??.*)/i
/^Hubot:?\s*(.*)(SUCCESS|NAILED IT.*)/i
/^Hubot:?\s*(.*) (ALL the .*)/
/^Hubot:?\s*gem whois (.*)/i

module.js:305
    throw err;
    ^
Error: Cannot find module '/home/leto/git/hubot/lib/sprintf'
    at Function._resolveFilename (module.js:326:11)
    at Function._load (module.js:271:25)
    at require (module.js:355:19)
    at Object.<anonymous> (/home/leto/git/hubot/scripts/vanity.coffee:4:13)
    at Object.<anonymous> (/home/leto/git/hubot/scripts/vanity.coffee:53:4)
    at Module._compile (module.js:411:26)
    at Object..coffee (/usr/local/lib/node_modules/coffee-script/lib/coffee-script.js:14:21)
    at Module.load (module.js:343:31)
    at Function._load (module.js:302:12)
    at require (module.js:355:19)
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,8 @@
     "xml2js": "0.1.x",
     "wolfram": "0.1.1",
     "soupselect": "0.2.0",
-    "htmlparser": "1.7.x"
+    "htmlparser": "1.7.x",
+    "sprintf": "0.1.1"
   },
 
   "directories":  {

--- a/src/scripts/vanity.coffee
+++ b/src/scripts/vanity.coffee
@@ -15,7 +15,7 @@
 #
 
 Path     = require "path"
-sprintf  = require(Path.resolve(Path.join(__dirname, "..", "lib", "sprintf"))).sprintf
+sprintf  = require("sprintf").sprintf
 
 countFollowers = (msg, ids, cb) ->
   counts = []
@@ -55,4 +55,3 @@ module.exports = (robot) ->
             msg.send output
         else
           msg.reply "Sorry, not right now. Twitter's returning a #{res.statusCode} error"
-

